app/services/provider_interface/application_data_export.rb

```diff
@@ -34,13 +34,13 @@ def self.call(application_choices:)
           'postcode' => application.application_form.postcode,
           'country' => application.application_form.country,
           'recruitment_cycle_year' => application.application_form.recruitment_cycle_year,
-          'provider_code' => application.offered_provider.code,
-          'accredited_provider_name' => application.accredited_provider&.name,
-          'accredited_provider_code' => application.accredited_provider&.code,
-          'course_code' => application.offered_course.code,
-          'site_code' => application.offered_site.code,
-          'study_mode' => application.offered_course.study_mode,
-          'start_date' => application.offered_course.start_date,
+          'provider_code' => application.current_provider.code,
+          'accredited_provider_name' => application.current_accredited_provider&.name,
+          'accredited_provider_code' => application.current_accredited_provider&.code,
+          'course_code' => application.current_course.code,
+          'site_code' => application.current_site.code,
+          'study_mode' => application.current_course_option.study_mode,
```

Comment:
The old code is looking at `current_course`, is there a difference between study mode on a course vs an option?

Reply:
Yes, `study_mode` on course can take the following values: `full_time`, `part_time`, `full_time_or_part_time` but `study_mode` on a course option can only be `full_time` or `part_time`. Considering the export deals with application choices, we should be using the course option one, as specific choices can only be full-time or part-time.